src/OSmOSE/frequency_scales/custom_frequency_scale.py

```diff
@@ -14,7 +14,7 @@ class CustomScale(AbstractScale):
         8000-22050 Hz : 10%  (e.g sr = 44100 Hz)
     """
 
-    frequencies: Tuple[int, int] = (22000, 100000)
+    frequencies: Tuple[int, int] = (1, 14000)
```

Comment:
indeed this modif should be undone , however could it be possible not to hardcode any numerical values here ? because (22000, 100000) is also somewhat arbitrary